sw/host/opentitanlib/src/dif/lc_ctrl.rs

```diff
@@ -93,6 +386,19 @@ impl From<[u8; 16]> for DifLcCtrlToken {
     }
 }
 
+impl From<[u32; 4]> for DifLcCtrlToken {
+    fn from(words: [u32; 4]) -> Self {
+        let mut bytes = [0u8; 16];
+        for i in 0..4 {
+            bytes[4 * i + 0] = (words[i] & 0xFF) as u8;
+            bytes[4 * i + 1] = ((words[i] >> 8) & 0xFF) as u8;
+            bytes[4 * i + 2] = ((words[i] >> 16) & 0xFF) as u8;
+            bytes[4 * i + 3] = ((words[i] >> 24) & 0xFF) as u8;
+        }
+        DifLcCtrlToken::from(bytes)
```

Comment:
If you wanted to be fancy here, you could try something like this:
```
        let bytes: [u8; 16] = words
            .iter()
            .map(|v| v.to_le_bytes())
            .flatten()
            .collect::<Vec<u8>>()
            .try_into() 
            .unwrap();
        DifLcCtrlToken::from(bytes)
```

Because the input is fixed size and the output is the same size in bytes, the `try_into` is going to succeed and, thus, we can simply unwrap the result.